src/components/Search/search-result.jsx

```diff
@@ -1,104 +1,94 @@
 import { Link } from 'gatsby';
 import { default as React } from 'react';
-import { Paragraph } from '@smallstep/step-ui';
+import { Heading } from '@smallstep/step-ui';
 import { makeStyles } from '@material-ui/styles';
 import {
   connectStateResults,
+  connectHits,
   Highlight,
-  Hits,
   Index,
   Snippet,
 } from 'react-instantsearch-dom';
-import { Box, ListItem, ListItemText } from '@material-ui/core';
+import { Box, Typography, ListItem, ListItemText } from '@material-ui/core';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) => ({
   resultsBorder: {
-    borderColor: '#D3D3D3',
-    marginBottom: -16,
-    marginTop: -2,
+    borderBottom: `1px solid ${theme.palette.accent.grey3}`,
+  },
+  link: {
+    textDecoration: 'none',
+    color: 'inherit',
   },
   hits: {
-    marginLeft: -30,
-    marginRight: 10,
     '& mark ': {
-      backgroundColor: '#84A8FF',
-    },
-    '& ul ': {
-      listStyle: 'none',
-    },
-    '& a:-webkit-any-link ': {
-      textDecoration: 'none',
-      color: 'inherit',
+      backgroundColor: theme.palette.accent.yellow,
     },
   },
-});
+}));
 
 const HitCount = connectStateResults(({ searchResults }) => {
   const hitCount = searchResults && searchResults.nbHits;
-  const theme = useStyles();
+  const classes = useStyles();
   return hitCount > 0 ? (
-    <Box borderBottom={1} textAlign={'right'} className={theme.resultsBorder}>
-      <Box mr={2}>
+    <Box textAlign={'right'} className={classes.resultsBorder} mb={2}>
+      <Typography variant="body1">
```

Comment:
I noticed that react was throwing warnings because `Paragraph` uses a `p` tag and it nests weirdly with some of the algolia stuff (and adds extra margin too). It ends up working best to just use the `Typography` component directly.